packages/replay/test/integration/errorSampleRate.test.ts

```diff
@@ -321,11 +322,10 @@ describe('Integration | errorSampleRate', () => {
     });
   });
 
-  it('has correct timestamps when error occurs much later than initial pageload/checkout', async () => {
+  it('keeps up to the last two checkout events xxx', async () => {
```

Comment:
Note: I tried to also add a test here that makes sure we clear out previous stuff after the previous checkout, but ran into many test issues here. So I ended up testing this in different places, mostly in addEvent.test as well as in eventBuffer.test.